### DISCUSSION TITLE
Picking file from Telescope is not opened in current window if it is a terminal buffer

### DISCUSSION THREAD

**Discussion by kyhung** [LazyVim/LazyVim#4568] [Q&A] [answered]
My expected behaviour is picking a file from Telescope will always be open in the current active window, no matter what type of buffer is active in the window. But after switching to LazyVim, the file picked from Telescope will be opened in the last active window with a non-terminal buffer. Is this the expected behaviour and can I change it? Thanks!

**Comment by dpetka2001** [comment, marked as answer]
Create a file `~/.config/nvim/lua/plugins/tele.lua` with contents
```lua
return {
  {
    "nvim-telescope/telescope.nvim",
    opts = {
      defaults = {
        get_selection_window = function()
          return 0
        end,
      },
    },
  },
}
```
That should give you what you want.

**Reply by kyhung** [reply]
Thank  you!
